app/assets/javascripts/discourse/app/templates/full-page-search.hbs

```diff
@@ -145,7 +145,7 @@
             <ComboBox
               @value={{this.sortOrder}}
               @content={{this.sortOrders}}
-              @onChange={{action (mut this.sortOrder)}}
+              @onChange={{action this.setSortOrder}}
```

Comment:
```suggestion
              @onChange={{this.setSortOrder}}
```

Reply:
I will never get used to these action changes :sweat_smile: